docs/background/plot_00_conceptual_intro.py

```diff
@@ -188,3 +194,7 @@
 #     `score` method, passing the predictors and the counts. The method first
 #     computes the rate $\lambda(t)$ using (2) and then the likelihood using
 #     (4). This method is used under the hood during optimization.
+
+# %%
+### More general GLMs
+# So far, we have focused on the relatively simple LNP model of spike generation, which is a special case of a GLM. The LNP model has some known shortcomings (Pillow et al 2008). For instance, LNP ignores things like refactory periods and other history-dependent features of spiking in a neuron. As we will show in other demos, such _spike history filters_ can be built into GLMs to give more accurate results. We will also show how, if you have recordings from a large _population_ of neurons simultaneously, you can build connections between the neurons into the GLM in the form of _coupling filters_. This can help answer the degree to which activity is driven primarily by the input X, or by network influences in the population. Some researcheers have used cascades of GLMs to model dendritic computation in individual neurons (Ujfalussy et al 2018). 
```

Comment:
I like this previewing what else they can find in our docs. For the papers, can you add citations using footnotes? (you can see an example in the current injection example).

Reply:
Yes I can do that. Incidentally I'm planning to adapt to all these super helpful review comments at _some_ point today/tonight. I'm getting ready for a big family trip and doing a bunch of other stuff simultaneously, but I will adjust PRs soon! 😅

Reply:
Your edits really improved the flow and info content of the notebook, I really like it.